server/tests/api_v4/authenticated/test_shamir_recovery_setup.py

```diff
@@ -0,0 +1,63 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import pytest
+
+from parsec._parsec import (
+    DateTime,
+    ShamirRecoveryBriefCertificate,
+    ShamirRecoveryShareCertificate,
+    ShamirRevealToken,
+    authenticated_cmds,
+)
+from tests.common import CoolorgRpcClients
+
+
+async def test_authenticated_shamir_recovery_setup_ok(
+    coolorg: CoolorgRpcClients, with_postgresql: bool
+) -> None:
+    if with_postgresql:
+        pytest.skip("TODO: postgre not implemented yet")
+    share = ShamirRecoveryShareCertificate(
+        coolorg.alice.device_id, DateTime.now(), coolorg.mallory.user_id, b"abc"
+    )
+    brief = ShamirRecoveryBriefCertificate(
+        author=coolorg.alice.device_id,
+        timestamp=DateTime.now(),
+        threshold=1,
+        per_recipient_shares={coolorg.mallory.user_id: 2},
+    )
+
+    setup = authenticated_cmds.v4.shamir_recovery_setup.ShamirRecoverySetup(
+        b"abc",
+        ShamirRevealToken(),
+        brief.dump_and_sign(coolorg.alice.signing_key),
+        [share.dump_and_sign(coolorg.alice.signing_key)],
+    )
+    rep = await coolorg.alice.shamir_recovery_setup(setup)
+    assert rep == authenticated_cmds.v4.shamir_recovery_setup.RepOk()
+
+    # TODO dump
+
+
+async def test_authenticated_shamir_recovery_setup_already_set(
+    coolorg: CoolorgRpcClients,
+) -> None:
+    pass
+
+
+async def test_authenticated_shamir_recovery_setup_invalid_certification(
+    coolorg: CoolorgRpcClients,
+) -> None:
+    pass
+
+
+async def test_authenticated_shamir_recovery_setup_invalid_data(
+    coolorg: CoolorgRpcClients, with_postgresql: bool
+) -> None:
+    if with_postgresql:
+        pytest.skip("TODO: postgre not implemented yet")
+    setup = authenticated_cmds.v4.shamir_recovery_setup.ShamirRecoverySetup(
+        bytes("abc", "utf-8"), ShamirRevealToken(), bytes("ijk", "utf-8"), [bytes("lmn", "utf-8")]
```

Comment:
```suggestion
        b"abc", ShamirRevealToken(), b"ijk", [b"lmn"]
```
Simpler notation, it's was done in the first test function on this file.

A specific reason to use `bytes` instead ?

Reply:
I forgot about this notation ^^ I'm a bit rusty in python